index.php

```diff
@@ -4,6 +4,66 @@
 include_once('functions.php');
 include_once('core/process/data.loader.php');
 
+function menuitems($menu, $level) {
```

Comment:
We don't have functions in the index.php
You should probably move it to functions.php until it gets refactored.
And menuitems to printMenuitems or something similar so we can understand what the function does more easily.